.github/workflows/test.yml

```diff
@@ -28,18 +74,27 @@ jobs:
       with:
         python-version: ${{ matrix.cfg.py }}
 
+    - name: Download the wheel
+      uses: actions/download-artifact@v4
+      with:
+        name: wheel
+        path: dist
+
+    - name: Install the wheel for Windows configuration
+      if: ${{ runner.os == 'Windows' }}
+      run: |
+        $wheel = Get-ChildItem -Path dist -Filter "periodictable*.whl" | Select-Object -First 1
+        python -m pip install $wheel.FullName
+      shell: pwsh
+
+    - name: Install the wheel for configurations other than Windows
+      if: ${{ runner.os != 'Windows' }}
+      run: python -m pip install dist/periodictable*.whl
+
     - name: Install Python dependencies
       run: |
-        python -m pip install --upgrade pip
-        python -m pip install wheel setuptools
         python -m pip install numpy scipy matplotlib pytest pytest-cov
```

Comment:
Scipy and matplotlib are required for tests and numpy is listed as a dependency. Since testing against installed wheel we should only need pytest and pytest-cov